dev/docker-compose.yaml

```diff
@@ -78,9 +78,7 @@ services:
       OIDC_USERINFO_URI: "https://keycloak.${BASE_DOMAIN:-localtest.me}/auth/realms/fEMR/protocol/openid-connect/userinfo"
       OIDC_TOKEN_URI: "https://keycloak.${BASE_DOMAIN:-localtest.me}/auth/realms/fEMR/protocol/openid-connect/token"
       OIDC_TOKEN_INTROSPECTION_URI: "https://keycloak.${BASE_DOMAIN:-localtest.me}/auth/realms/fEMR/protocol/openid-connect/token/introspect"
-      # TODO change to INTERNAL_FHIR_API
-      # TODO switch to external API (requiring auth)
-      MAP_API: "http://fhir-internal:8080/fhir/"
+      MAP_API: 'https://fhirwall.${BASE_DOMAIN:-localtest.me}/fhir/'
```

Comment:
I'll note, this overdue change was necessary to pick up FHIR mutations initiated in fEMR.